FirebaseAuth/Sources/AuthProvider/Phone/FIRPhoneAuthProvider.m

```diff
@@ -593,6 +595,16 @@ - (void)verifyClientAndSendVerificationCodeToPhoneNumber:(NSString *)phoneNumber
  */
 - (void)verifyClientWithUIDelegate:(nullable id<FIRAuthUIDelegate>)UIDelegate
                         completion:(FIRVerifyClientCallback)completion {
+  // Remove the simulator check below after FCM supports APNs in simulators
+  if (@available(iOS 17, *)) {
```

Comment:
How will this change make any difference for iOS 16 simulator users?

Reply:
Ops, wrong commit. Just fixed.